pyproject.toml

```diff
@@ -318,3 +318,17 @@ testpaths = ["lib"]
 addopts = [
     "--import-mode=importlib",
 ]
+
+# This build is just for testing right now.
+[tool.cibuildwheel.pyodide]
+build-frontend = "build; args: --exports whole_archive"
+test-requires = "pytest"
+test-command = "pytest -p no:cacheprovider --pyargs matplotlib"
+[tool.cibuildwheel.pyodide.config-settings]
+install-args = "--tags=data,python-runtime,runtime,tests"
+[tool.cibuildwheel.pyodide.environment]
+# Exceptions are needed for pybind11:
+# https://github.com/pybind/pybind11/pull/5298
+CFLAGS = "-fexceptions"
+CXXFLAGS = "-fexceptions"
+LDFLAGS = "-fexceptions"
```

Comment:
Someone was suggesting recently that we ought to make this the default.

Reply:
Yes, Henry was suggesting it in our Discord server, IIRC, through which https://github.com/pybind/pybind11/pull/5298 came up (as linked in the code comment above)